client/src/modules/depots/templates/label.tmpl.html

```diff
@@ -0,0 +1,10 @@
+<div class="ui-grid-cell-contents">
+  <a data-method="stock-inventories"
+    ui-sref="stockInventories({ filters : [
+      { key : 'depot_uuid', value : row.entity.uuid, displayValue : row.entity.text },
+      { key : 'period', value : 'allTime', displayValue : 'PERIODS.ALL_TIME' },
+      { key : 'limit', value : 1000 }
```

Comment:
I think you can put `cacheable:false` on all these filters to make sure that the FilterService cdoesn't automatically save them and override whatever the previous user's values were.

Reply:
See [FilterService.js#L149](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/services/FilterService.js#L149) for implementation.

Reply:
I tried to put `cacheable:false` but it seems like there is no effect 😢